tests/playwright/src/fixtures/electron-app.ts

```
@@ -19,23 +19,91 @@ import { dirname, resolve } from 'node:path';
 
-export const TIMEOUTS = {
-  FIRST_WINDOW: 120_000,
-  PAGE_LOAD: 90_000,
-  WINDOW_EVENT: 120_000,
-  NON_DEVTOOLS_WINDOW: 60_000,
-  RETRY_DELAY: 1_000,
-  MAX_RETRIES: 3,
-} as const;
+export const test = base.extend<ElectronFixtures>({
+  // eslint-disable-next-line no-empty-pattern
+  electronApp: async ({}, use): Promise<void> => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)